controllers/kogitoserverlessworkflow_controller.go

```diff
@@ -18,7 +18,7 @@ package controllers
 
 import (
 	"context"
-	serverlessv1alpha1 "github.com/davidesalerno/kogito-serverless-operator/api/v1alpha1"
+	serverlessv08 "github.com/davidesalerno/kogito-serverless-operator/api/v08"
```

Comment:
https://github.com/kiegroup/kogito-operator/pull/1281

Reply:
Are you suggesting to switch from serverlessv08 to something like apiv08?

Reply:
lol sry, I don't know how this link ended here
see my comment below.